flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/OperatorChain.java

```diff
@@ -786,17 +856,33 @@ private <OUT, OP extends StreamOperator<OUT>> OP createOperator(
     private <IN, OUT> WatermarkGaugeExposingOutput<StreamRecord<IN>> wrapOperatorIntoOutput(
             OneInputStreamOperator<IN, OUT> operator,
             StreamTask<OUT, ?> containingTask,
+            StreamConfig prevOperatorConfig,
             StreamConfig operatorConfig,
             ClassLoader userCodeClassloader,
-            OutputTag<IN> outputTag) {
+            OutputTag<IN> outputTag,
+            boolean shouldAddMetric) {
```

Comment:
```suggestion
            boolean shouldAddMetricForPrevOperator) {
```

Reply:
Thanks for the suggestion. It is fixed now.